engine/swarm/admin_guide.md

```diff
@@ -263,13 +263,25 @@ manager node of a single-node swarm. It discards swarm membership information
 that existed before the loss of the quorum but it retains data necessary to the
 Swarm such as services, tasks and the list of worker nodes.
 
-### Joining a previously failed node
-
-If a node becomes unavailable, it cannot communicate with the rest of the swarm
-and its workload is redistributed among the other nodes.
-If access to that node is restored, it will join the swarm automatically, but it
-will join with no workload because the containers it was assigned have been
-reassigned. The node will only receive new workloads when the swarm is rebalanced.
-To force the swarm to be rebalanced, you can
-[update](../reference/commandline/service_update/) or
-[scale](../reference/commandline/service_scale/) the service.
+### Re-balancing a service after joining a new or previously failed node
+
+When you add a new node to a swarm, or a node re-joins after it has been
```

Comment:
I'm not in love with that, because the verb in the CLI is 'join' rather than 'connect'. In fact, I should probably change the 'add' to 'join' for consistency. I could probably just change 're-join' to 'join'. WDYT?